### ISSUE TITLE
Are there exceptions for "without requiring specific timings for individual keystrokes" (SC 2.1.1)?

### ISSUE THREAD

**Issue by JAWS-test** [w3c/wcag#1539]
> [Success Criterion 2.1.1 Keyboard (Level A):](https://www.w3.org/WAI/WCAG21/Understanding/keyboard.html) All functionality of the content is operable through a keyboard interface **without requiring specific timings for individual keystrokes**

With the HTML select tag, but also with the [ARIA equivalent](https://w3c.github.io/aria-practices/examples/combobox/combobox-select-only.html), the time interval between the input of characters is relevant for whether they are interpreted as a new input or a continuation of the already made input. Would this be a violation of 2.1.1?

I also reach the list entries with the arrow keys, but if I have a lot of list entries, navigation by entering characters can be helpful. It is hard to imagine an alternative of operation, except maybe:
* AT for keyboard users collects user input and then sends itself to the browser with a short interval or
* each character input is considered a continuation of the previous one and only with a special access key a new input can be started

**Comment by alastc** [comment]
As the arrow keys can be used without timing, I don't see how the select tag approach would fail 2.1.1

**Comment by patrickhlauke** [comment]
agree, the ability to type a few characters (quickly enough) to jump directly to a select's option is not the only way in which a select is operated. it's a shortcut/enhancement. keyboard users can still operate it without this (yes, it may be more tedious, but "don't make it tedious" is not a WCAG requirement per se).

**Comment by alastc** [comment]
> but "don't make it tedious" is not a WCAG requirement per se).

Some might argue that '**do** make it tedious' is a WCAG approach ;-)

**Comment by alastc** [comment]
@JAWS-test does this answer your question? I'm not seeing a change to make, so I'd like to close the issue unless there is further information or a suggestion for a change to make.
